src/lang/Messages.properties

```diff
@@ -1625,6 +1625,10 @@ report.unknown.error=Failed to write to file {0}, see log for detail
 report.write.overwrite.dialog.title=Existing File
 report.write.overwrite.dialog.message= The selected file already exists. Do you want to replace it?
 report.write.permission.dialog.title=Permissions Failure
+report.write.diskspace.warning.dialog.title=Low Disk Space
+report.write.diskspace.warning.dialog.message=There may not be enough disk space to continue the operation, would you like to continue?
```

Comment:
We should tell how much space we check, so the user can judge better if it's enough or not.